django/db/migrations/operations/fields.py

```diff
@@ -292,6 +300,24 @@ def state_forwards(self, app_label, state):
                     [self.new_name if n == self.old_name else n for n in together]
                     for together in options[option]
                 ]
+        # Fix to_fields to refer to the new field.
+        model_tuple = app_label, self.model_name_lower
```

Comment:
Is it more readable to include the surrounding parenthesis? I'm not sure if this has come up before.